src/simulation/monsters/low/TzHaarKet.ts

```diff
@@ -0,0 +1,30 @@
+import LootTable from '../../../structures/LootTable';
+import SimpleMonster from '../../../structures/SimpleMonster';
+
+const TzHaarKetTable = new LootTable({ limit: 128 })
+	.oneIn(512, 'Tzhaar-ket-om')
+	.oneIn(512, 'Toktz-ket-xil')
+	.oneIn(512, 'Obsidian cape')
+	.oneIn(2000, 'Obsidian helmet')
+	.oneIn(2000, 'Obsidian platebody')
+	.oneIn(2000, 'Obsidian platelegs')
+
+	/* Materials */
+	.add('Uncut sapphire', 1, 5)
```

Comment:
i would say these are probably wieghted a little higher than this but its tough to be sure cuz the wiki doesnt have exact rates.  At the very least can you place a TODO here saying to check back and see if the wiki has exact rates so we dont forget